src/plugins/dev_tools/public/application.tsx

```diff
@@ -84,6 +86,7 @@ function DevToolsWrapper({ devTools, activeDevTool, updateRoute }: DevToolsWrapp
               setHeaderActionMenu: () => undefined,
               // TODO: adapt to use Core's ScopedHistory
               history: {} as any,
+              theme$,
```

Comment:
@elastic/kibana-stack-management had to adapt `DevToolsWrapper` to pass down the `theme$` as you're using core's `AppMount` interface.